index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Captain train - Open source</title>
+    <link rel="shortcut icon" href="assets/images/favicon.ico" id="favicon" />
+    <link rel="stylesheet" type="text/css" href="assets/styles/styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+        <div class="page">
+            <header class="header">
+                <div class="header__wrapper">
+                    <h1><a class="header__logo" href="http://captaintrain.github.io" title="Captain&nbsp;Train"></a></h1>
+                </div>
+            </header>
+            <main class="content">
+                <div class="content__wrapper">
+                    <article class="content__article">
+                        <h2>Stations Studio</h2>
+                        <p>Stations allows to visualize the stations of <a href="https://github.com/captaintrain/stations">https://github.com/captaintrain/stations</a>.</p>
+                        <p class="right">
+                            <a class="button" href="stations-studio/">Open the app</a>
+                        </p>
+                    </article>
+                </div>
+            </main>
+            <footer class="footer">
+                <ul class="footer__links">
+                    <li><a class="footer__links__link"  href="https://github.com/captaintrain/">Github</a></li>
```

Comment:
Could you please change it to GitHub, please? (Notice the uppercase H, which is how GitHub brands itself in marketing documents.)